testdata/reference_values.toml

```diff
@@ -0,0 +1,3 @@
+binary_256sha_digests = ["322527c0260e25f0e9a2595bd0d71a52294fe2397a7af76165190fd98de8920d"]
+want_build_cmds = true
+builder_image_256sha_digests = ["9e2ba52487d945504d250de186cb4fe2e3ba023ed2921dd6ac8b97ed43e76af9", "y"]
```

Comment:
```suggestion
builder_image_sha256_digests = ["9e2ba52487d945504d250de186cb4fe2e3ba023ed2921dd6ac8b97ed43e76af9", "y"]
```

Reply:
We could also think of the the following format. Let's think and discuss this separately:

```
builder_image_igests = ["sha256:9e2ba52487d945504d250de186cb4fe2e3ba023ed2921dd6ac8b97ed43e76af9", "y"]
```

Reply:
Sounds like a good idea, if we have other digests! Yes, let's discuss this seperately, but leave it for now.